src/cli/datasets/upload.py

```diff
@@ -47,5 +47,5 @@ def upload(path: str, bucket: str, prefix: str):
     s3 = default_session().client("s3")
     for config in tqdm(DATASET_REGISTRY.values()):
         upload_directory(
-            s3, path / config.name(), bucket, f"{prefix}/{config.name()}"
+            s3, path + '/' + config.name(), bucket, f"{prefix}/{config.name()}"
```

Comment:
Why is this needed?

Reply:
Since both path and config.name() are of type str, "path / config.name()" will raise an error.